static/docs/get-started/install.md

```diff
@@ -1,95 +0,0 @@
-# Install
-
-There are a few recommended ways to install DVC: OS-specific package/installer,
-`pip`, `conda` (Anaconda), and Homebrew. (Depending on your OS, some of these
-may be not available.)
-
-To install DVC from `pip` (see instructions for other options below), run:
-
-```dvc
-$ pip install dvc
-```
-
-> If you installed DVC via `pip`, depending on the
-> [remote](/doc/command-reference/remote) type you plan to use you might need to
-> install optional dependencies: `[s3]`, `[ssh]`, `[gs]`, `[azure]`, and
-> `[oss]`; or `[all]` to include them all. The command should look like this:
-> `pip install "dvc[s3]"`. This installs `boto3` library along with DVC to
-> support Amazon S3 storage.
-
-The easiest option, self-contained binary packages (or Windows installer), is
-available by using the big "Download" button on the [home page](/). You can also
-get them [here](https://github.com/iterative/dvc/releases/).
-
-> For a POSIX-like command line terminal in Windows, we recommend
```

Comment:
move this to Windows install if it's not there yet?

Reply:
It is there already.

Reply:
👍